msteams-platform/concepts/deploy-and-publish/appsource/prepare/purchase-webapps.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Include a SaaS offer for your apps
+description: Describes the process of creating SaaS offers and the purchase experience for users subscribing to apps with SaaS offers.
+ms.author: surbhigupta
+ms.topic: conceptual
+localization_priority: Normal 
+---
+
+# Include a SaaS offer for your apps
+
+In the Teams store, you have apps that are free to install or are available as software-as-a-service (SaaS). If you want your app to be available as SaaS, then you must follow the complete process of planning a SaaS offer, creating a SaaS offer in the Partner Center's Microsoft commercial marketplace, linking the SaaS offer in your manifest through the Teams Developer Portal, publishing the app to the Teams store, and enabling the SaaS offer for your app.
+
+> [!NOTE]
+> It is recommended to configure your SaaS offer before publishing your app to the Teams store. However, if your app is already published and you now want to change it to SaaS, then you must republish your app after configuring the SaaS offer for your app.
+
+## Plan your SaaS offer
+
+To plan your SaaS offer, see [How to plan a SaaS offer for the commercial marketplace](/azure/marketplace/plan-saas-offer).
+
+> [!NOTE]
+> When you include a SaaS offer for your app, you manage and pay for the infrastructure required to support your customers' use of your offer.
+
+## Create a SaaS offer
+
+> [!NOTE]
+> You must have a [Partner Center account](create-partner-center-dev-account.md) to create a SaaS offer.
+
+To configure plans and pricing details for your app, create a SaaS offer in the [Microsoft commercial marketplace](/azure/marketplace/create-new-saas-offer).
+
+## Link your SaaS offer through Teams Developer Portal
+
+In the [Developer Portal](~/build-and-test/teams-developer-portal.md), provide your **Publisher ID** and **Offer ID**. The manifest is updated automatically. The `subscriptionOffer` property is added to the manifest.
+
+**To link your SaaS offer**
+
+1. Go to [Developer Portal](https://aka.ms/dev-portal), and select **Apps**.
+2. On the **Apps** page, select the app for which you want to enable the SaaS offer.
+3. Select **Plans and Pricing** in the left pane. (*Screenshot / mocks required*)
+4. In the **Plans and Pricing** page, enter your **Publisher ID** and **Offer ID**.
+
+    > [!NOTE]
+    > You can get your publisher ID and Offer ID from the Partner Center.
+1. Select **Preview** to view the details of the SaaS offer, and then select **Save**.
+1. Under **Distribute** in the left pane, select **Manifest**. Following property is added to the manifest:
+
+    ```json
```

Comment:
Per developer: Uha, we can add some Note at step #6 (Under Distribute in the left pane, select Manifest.) The "manifestVersion" needs to be "devPreview", "v1.10" or above. Otherwise it will fail at validation with error (Proerty "subscriptionOffer" has not been defined...) during publish app.

Reply:
Done. Added to a Prerequisites section.